src/unix/async.c

```diff
@@ -231,6 +231,18 @@ int uv__async_start(uv_loop_t* loop, struct uv__async* wa, uv__async_cb cb) {
 }
 
 
+int uv__async_fork(uv_loop_t* loop, struct uv__async* wa) {
+  if (wa->io_watcher.fd == -1) /* never started */
+    return 0;
+
+  uv__async_stop(loop, wa);
+
+  uv__async_init(wa);
+
+  return uv__async_start(loop, &loop->async_watcher, uv__async_event);
+}
```

Comment:
This is going to need a rebase, the fields are `loop->async_io_watcher` and `loop->async_wfd` now, and the `uv__async` type has been removed.  I removed `uv__async_init()` but you could consider reintroducing it.